@vadmeste you may want to fix the tests

```
--- FAIL: TestUserAgent (0.00s)
    update_test.go:183: Test 1: expected: MinIO (linux; amd64; mode-server-fs; source) MinIO/DEVELOPMENT.GOGET MinIO/DEVELOPMENT.GOGET MinIO/DEVELOPMENT.GOGET, got: MinIO (linux; amd64; mode-server-fs; source) MinIO/DEVELOPMENT.GOGET MinIO/DEVELOPMENT.GOGET MinIO/DEVELOPMENT.GOGET CPU (total_cpus:1, total_cores:2; vendor:GenuineIntel; family:6; model:106; stepping:6; model_name:Intel(R) Xeon(R) Platinum 8370C CPU @ 2.80GHz)
    update_test.go:183: Test 2: expected: MinIO (linux; amd64; mode-server-xl; dcos; source) MinIO/DEVELOPMENT.GOGET MinIO/DEVELOPMENT.GOGET MinIO/DEVELOPMENT.GOGET MinIO/universe-mesos-1111, got: MinIO (linux; amd64; mode-server-xl; dcos; source) MinIO/DEVELOPMENT.GOGET MinIO/DEVELOPMENT.GOGET MinIO/DEVELOPMENT.GOGET MinIO/universe-mesos-1111 CPU (total_cpus:1, total_cores:2; vendor:GenuineIntel; family:6; model:106; stepping:6; model_name:Intel(R) Xeon(R) Platinum 8370C CPU @ 2.80GHz)
    update_test.go:183: Test 3: expected: MinIO (linux; amd64; mode-server-xl; kubernetes; source) MinIO/DEVELOPMENT.GOGET MinIO/DEVELOPMENT.GOGET MinIO/DEVELOPMENT.GOGET, got: MinIO (linux; amd64; mode-server-xl; kubernetes; source) MinIO/DEVELOPMENT.GOGET MinIO/DEVELOPMENT.GOGET MinIO/DEVELOPMENT.GOGET CPU (total_cpus:1, total_cores:2; vendor:GenuineIntel; family:6; model:106; stepping:6; model_name:Intel(R) Xeon(R) Platinum 8370C CPU @ 2.80GHz)
FAIL
```

--- a/cmd/update_test.go
+++ b/cmd/update_test.go
@@ -179,7 +179,7 @@ func TestUserAgent(t *testing.T) {
 		if IsDocker() {
 			expectedStr = strings.ReplaceAll(expectedStr, "; source", "; docker; source")
 		}
-		if str != expectedStr {
+		if !strings.Contains(str, expectedStr) {
 			t.Errorf("Test %d: expected: %s, got: %s", i+1, expectedStr, str)
 		}
 		globalIsCICD = sci

